src/class/msc/msc.h

```diff
@@ -255,7 +255,7 @@ typedef struct TU_ATTR_PACKED
 
   uint8_t : 3;
   uint8_t disable_block_descriptor : 1;
-  uint8_t : 0;
+  uint8_t : 4;
```

Comment:
hmm, there is probably other places that have `:0` padding somewhere in the codebase, I didn't know this is not in the specs :facepalm:

Reply:
Yes there are many zero width bitfields :) I've also reported this bug to IAR

Reply:
```
an integral constant expression with a value greater or equal to zero. When greater than zero, this is the number of bits that this bit field will occupy. The value zero is only allowed for nameless bitfields and has special meaning: it specifies that the next bit field in the class definition will begin at an allocation unit's boundary.
```

I checked a bit further, and indeed it is part of the specs (not entire sure which specs version it belongs to). Yes, it is good idea to report this to IAR.